docs/source/quickstart.md

```diff
@@ -358,34 +323,25 @@ You can also retrieve the individual vectorial impulses:
 >>> tuple(val.decompose([u.km, u.s]) for val in hoh[1])
 (<Quantity 15729.741535747102 s>, <Quantity [ 0.        , 1.41999995, 0.        ] km / s>)
 ```
-```{eval-rst}
-.. _Hohmann: https://en.wikipedia.org/wiki/Hohmann_transfer_orbit
-.. _bielliptic: https://en.wikipedia.org/wiki/Bi-elliptic_transfer
+
 
 To actually retrieve the resulting ``Orbit`` after performing a maneuver, use
-the method :py:meth:`~poliastro.twobody.orbit.Orbit.apply_maneuver`:
-```
+the method {py:meth}`~poliastro.twobody.orbit.Orbit.apply_maneuver`:
+
 
 ```python
 >>> ss_f = ss_i.apply_maneuver(hoh)
 >>> ss_f
 36000 x 36000 km x 0.0 deg (GCRS) orbit around Earth (♁)
 ```
 
-```{eval-rst}
-More advanced plotting: :py:class:`~poliastro.plotting.OrbitPlotter` objects
-----------------------------------------------------------------------------
 
-We previously saw the :py:func:`poliastro.plotting.plot` function to easily
-plot orbits. Now we'd like to plot several orbits in one graph (for example,
-the maneuver we computed in the previous section). For this purpose, we
-have :py:class:`~poliastro.plotting.OrbitPlotter` objects in the
-:py:mod:`~poliastro.plotting` module.
-```
+#### More advanced plotting: {py:class}`~poliastro.plotting.OrbitPlotter` objects
```

Comment:
This should be a level 3 header, just like the previous one. Also, I realized the `OrbitPlotter` doesn't exist.

```suggestion
### More advanced plotting: `OrbitPlotter*` objects
```